ami/trainers/i_jepa_latent_visualization_trainer.py

```diff
@@ -0,0 +1,217 @@
+import time
+from functools import partial
+from pathlib import Path
+from typing import Literal
+
+import torch
+import torch.nn.functional as F
+import torchvision.transforms.v2.functional as torchvisF
```

Comment:
`torchvisF`って記述にすごい違和感　リサイズの処理はちょっと長いけど`torchvision.transforms.v2.functional.resize(image_batch, image_size)`って形で呼び出したい
どうせここ以外で`torchvision.transforms.v2.functional`の機能呼び出してる所ないし
```suggestion
import torchvision
```

Reply:
`import torchvision.transforms.v2.functional`とすれば大丈夫かな。 ここまで書かないと補完が効かない